src/edge_kernels/MomentumEdgeSolverAlg.C

```diff
@@ -130,8 +144,8 @@ MomentumEdgeSolverAlg::execute()
         }
       }
 
-      const DblType pecfac = pecletFactor.get(edge, 0);
-      const DblType om_pecfac = 1.0 - pecfac;
+      DblType pecfac = pecletFactor.get(edge, 0);
```

Comment:
could we move this down a bit? Closer to your upwinding switch comment since that's closer to where it is used/needed?

Reply:
done